aikau/src/main/resources/alfresco/core/ResizeMixin.js

```diff
@@ -61,7 +61,9 @@ define(["dojo/_base/declare",
          // Fire a custom event to let contained objects know that the node has been resized.
          this.throttledPublish(this.alfResizeNodeTopic, {
             node: resizedNode
-         }, true);
+         }, true, false, {
+            timeoutMs: 100
```

Comment:
We're the architects of the framework, who make informed decisions based upon stability/performance considerations. If we do make this configurable, then we'll also need to somehow make the common functionUtils class be configurable, as it's in there that the values of 100, 1000 and 10000 are statically defined as appropriate timeouts for repeating functions, and ResizeMixin relies upon domUtils which relies upon functionUtils to achieve this. It does not seem very viable to do, but if you like we can try and work out a way of this happening.